observatory-platform/requirements.txt

```diff
@@ -34,13 +34,19 @@ json_lines>=0.5.0,<1 # Reading, including jsonl.gz
 # HTTP requests and URL cleaning
 requests>=2.25.0,<3
 tldextract>=3.1.1
+aiohttp
+
 
 # SFTP
 pysftp>=0.2.9<1
 paramiko>=2.7.2,<3
 
 # Utils
 natsort>=7.1.1,<8
+backoff==1.11.*
```

Comment:
Could you put a looser definition for the backoff version? e.g. `backoff>=1,<2`, since the observatory platform can be depended on by downstream projects, it shouldn't be too strict about library versions.